1/contrib/openshift/configuration/jobs/OpenShift Sample/config.xml

```diff
@@ -13,29 +13,61 @@
   <builders>
 
     <com.openshift.jenkins.plugins.pipeline.OpenShiftScaler>
-       <replicaCount>0</replicaCount>
+      <apiURL>https://openshift.default.svc.cluster.local</apiURL>
+      <depCfg>frontend</depCfg>
+      <namespace>test</namespace>
+      <replicaCount>0</replicaCount>
+      <authToken></authToken>
     </com.openshift.jenkins.plugins.pipeline.OpenShiftScaler>
 
     <com.openshift.jenkins.plugins.pipeline.OpenShiftBuilder>
+      <apiURL>https://openshift.default.svc.cluster.local</apiURL>
+      <bldCfg>frontend</bldCfg>
+      <namespace>test</namespace>
+      <authToken></authToken>
       <checkForTriggeredDeployments>true</checkForTriggeredDeployments>
     </com.openshift.jenkins.plugins.pipeline.OpenShiftBuilder>
 
     <com.openshift.jenkins.plugins.pipeline.OpenShiftDeploymentVerifier>
+      <apiURL>https://openshift.default.svc.cluster.local</apiURL>
+      <depCfg>frontend</depCfg>
+      <namespace>test</namespace>
+      <replicaCount>0</replicaCount>
+      <authToken></authToken>
     </com.openshift.jenkins.plugins.pipeline.OpenShiftDeploymentVerifier>
 
     <com.openshift.jenkins.plugins.pipeline.OpenShiftScaler>
+      <apiURL>https://openshift.default.svc.cluster.local</apiURL>
+      <depCfg>frontend</depCfg>
+      <namespace>test</namespace>
       <replicaCount>1</replicaCount>
+      <authToken></authToken>
       <verifyReplicaCount>true</verifyReplicaCount>
     </com.openshift.jenkins.plugins.pipeline.OpenShiftScaler>
     
     <com.openshift.jenkins.plugins.pipeline.OpenShiftServiceVerifier>
+      <apiURL>https://openshift.default.svc.cluster.local</apiURL>
+      <svcName>frontend</svcName>
+      <namespace>test</namespace>
+      <authToken></authToken>
     </com.openshift.jenkins.plugins.pipeline.OpenShiftServiceVerifier>
 
     <com.openshift.jenkins.plugins.pipeline.OpenShiftImageTagger>
+      <apiURL>https://openshift.default.svc.cluster.local</apiURL>
+      <namespace>test</namespace>
+      <testTag>latest</testTag>
+      <prodTag>prod</prodTag>
+      <testStream>origin-nodejs-sample</testStream>
+      <prodStream>origin-nodejs-sample</prodStream>
```

Comment:
No, this was just splitting our the images stream and tag name into separate parameters.

Yeah, source and destination would be better.  I would suggest doing that post 3.2, as it would require a rebuild of the plugin, which would delay delivery.